parcels/particleset.py

```diff
@@ -382,6 +382,69 @@ def show(self, **kwargs):
             print('Plot saved to '+savefile+'.png')
             plt.close()
 
+    # def density(self, field, particle_val=None):
```

Comment:
Is there a reason for leaving all of this commented out in the code? Seems an old version of the newer density() function below